code/game/machinery/kitchen/microwave.dm

```diff
@@ -2,24 +2,26 @@
 	name = "microwave"
 	icon = 'icons/obj/kitchen.dmi'
 	icon_state = "mw"
-	layer = 2.9
-	density = 1
-	anchored = 1
-	use_power = 1
+	layer = 2.9 // ARBITRARY NUMBERS
+	density = TRUE
+	anchored = TRUE
+	use_power = TRUE
 	idle_power_usage = 5
 	active_power_usage = 100
 	flags = OPENCONTAINER | NOREACT
 	circuit = /obj/item/weapon/circuitboard/microwave
-	var/operating = 0 // Is it on?
+	var/operating = 0 // Is it on? gee thanks i needed that clarified
 	var/dirty = 0 // = {0..100} Does it need cleaning?
 	var/broken = 0 // ={0,1,2} How broken is it???
-	var/global/list/datum/recipe/available_recipes // List of the recipes you can use
+	// typecache and global these
+	var/global/list/datum/recipe/available_recipes // List of the recipes you can use 
 	var/global/list/acceptable_items // List of the items you can put in
 	var/global/list/acceptable_reagents // List of the reagents you can put in
-	var/global/max_n_of_items = 0
+	var/global/max_n_of_items = 0 // why the fuck is this a global
 
 
 // see code/modules/food/recipes_microwave.dm for recipes
+// see code/modules/hell/occupants.dm for coder

```

Comment:
This whole pull is like 90% pointless rant comments.

Reply:
a good rant helps mark shitcode, which 90% of this file is